api-docs/apiLive.htm

```diff
@@ -35558,7 +35616,769 @@ <h2 class="flybar-button">Template</h2>
+					</table>
+				</div>
+			</div>
+			
+			<a id="client_transaction" name="client_transaction" class="old-syle-anchor">&nbsp;</a>
+			<div class="method-section">
+				<div class="method-description">
+					<h3>Client Transaction</h3>
+					<p>Client Charge transactions are get created when a client pays/waives off client charge.this client charge transaction shows
+						<br>the unit of amount paid as client charge.
+					</p>
+					<table class=matrixHeading>
+						<tr class="matrixHeadingBG">
+							<td><div class="mifosXHeading2">Field Descriptions</div></td>
+						</tr>
+						<tr class=alt>
+							<td>transactionId</td>
+						</tr>
+						<tr>
+							<td class=fielddesc>This ia an unique Id associate with client charge Transaction.
+							</td>
+						</tr>
+						<tr class=alt>
+							<td>clientId</td>
+						</tr>
+						<tr>
+							<td class=fielddesc>clientId on which client transaction got created</td>
+						</tr>
+						<tr class=alt>
+							<td>isReverse</td>
+						</tr>
+						<tr>
+							<td class=fielddesc>Indicates whether this client charge transaction is reversed or not.
+							</td>
+						</tr>
+						<tr class=alt>
+							<td>amountPaid</td>
+						</tr>
+						<tr>
+							<td class=fielddesc>amount paid as client charge</b>.
+							</td>
+						</tr>
+						<tr class=alt>
+							<td>currency code</td>
+						</tr>
+						<tr>
+							<td class=fielddesc>currency type</td>
+						</tr>
+						<tr class=alt>
+							<td>transaction date</td>
+						</tr>
+						<tr>
+							<td class=fielddesc>date on which transaction happened</td>
+						</tr>
+					
+					</table>
+				</div>
+			</div>
+			<a id="list_clientTransactions" name="list_clientTransactions" class="old-syle-anchor">&nbsp;</a>
+			<div class="method-section">
+				<div class="method-description">
+					<h4>List client Transactions</h4>
+					<p>The <i>list</i> capability of client charge transaction can support <b>pagination</b> and <b>sorting</b>.</p>
+					<h5>Mandatory Arguments</h5>
+					<dl class="argument-list">
+						<dt>offset</dt>
+						<dd>
+							Integer <span>Mandatory</span>, defaults to 0
+						</dd>
+						<dd>Indicates the result from which pagination starts</dd>
+
+						<dt>limit</dt>
+						<dd>
+							Integer <span>Mandatory</span>, defaults to 200
+						</dd>
+						<dd>Restricts the size of results returned. To override the default and return all entries you must explicitly pass a non-positive integer value for limit e.g. limit=0, or limit=-1</dd>
+            </dl>
+           </dl> 
+					<p>Example Requests:</p>				
+					<div class=apiClick>clients/189/transactions</div>
+					<div class=apiClick>clients/189/transactions?offset=10&limit=50</div>				
+				</div>
+				<div class="method-example">
+					<code class="method-declaration">
+GET https://DomainName/api/v1/clients/{clientId}/transactions?limit=5&offset=0
+					</code>
+					<code class="method-request">
+GET clients/226/transactions?limit=5&offset=0
```

Comment:
Documentation says that client transactions are paginated. however there are no pagination related changes in read platform service